tools/collector/edgex/collect.go

```diff
@@ -17,13 +17,20 @@ limitations under the License.
 package edgex
 
 import (
+	"bufio"
+	"os"
+	"strings"
+
 	"github.com/sirupsen/logrus"
 )
 
 var (
 	collectLog           *logrus.Entry
 	branchesURL          = "https://github.com/edgexfoundry/edgex-compose/branches/all"
 	extractVersionRegexp = `branch="(.*?)"`
+	singleArchPath       = "./config/singlearch_imagelist.txt"
+	multiArchPath        = "./config/multiarch_imagelist.txt"
+	repoName             = "openyurt/"
```

Comment:
The repo name should be read from the makefile to make debugging easier.

Reply:
ok,I will change it.